File: sunbin/day18/002_대문자와소문자.js
// https://school.programmers.co.kr/learn/courses/30/lessons/120893

function solution(my_string) {
  let answer = '';

  // for of, if ~ else
  for (const i of my_string) {
    if (i === i.toLowerCase()) {
      answer += i.toUpperCase();
    } else {
      answer += i.toLowerCase();
    }
  }

  // 같지만 다른
  for (let c of my_string) answer += c === c.toLowerCase() ? c.toUpperCase() : c.toLowerCase();

  return answer
}

File: sunbin/day18/python/002_대문자와소문자.py
# https://school.programmers.co.kr/learn/courses/30/lessons/120893

def solution(my_string):
    answer = ''
    for i in my_string:
        if i.islower():
          answer += i.upper()
        else:
          answer += i.lower()


# 다른풀이: Str.swapcase()

def solution(my_string):
    return my_string.swapcase()

